common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -134,6 +137,111 @@ def _create(cls, target_class, **kwargs):
                 return new_course
 
 
+class SampleCourseFactory(CourseFactory):
```

Comment:
Does the underlying CourseFactory use a mocked mongo as a datastore? That would be preferred (except for tests where using mongo is part of what's being tested)